### PR TITLE
Initial commit of attendance management system

### PR DESCRIPTION
**Pull request by MugAbdoul**

### CODE REVIEW

**Comment by MugAbdoul** [comment]
Implemented the Attendance Management System. Please review the changes to the database schema, sample data, and the attendance calculation procedure.

**Comment by Niphilbert** [comment]
Great job team! The project is very polished, and everything looks and works wonderfully. let's keep it up!

**Comment by NaomeJoyeuse** [comment]
Calculate_Attendance_Statistics procedure here nicely aligns with our approach we discussed  showing clear attendance stats per employee for chosen dates

**Comment by ruhanikaalex** [comment]
Proud of this team and the work we’ve done.

**Comment by Fabrice6-tech** [comment]
Wonderful! Am great and happy to see all  works here are giving , clear and understandable.. let’s keep working hard 💪

File: 1.png
Comment (Mikamahoro) [inline 1.png:1]: Looks great! The attendance calculation works as expected

**Comment by uwamahoro123** [comment]
👩‍💻 Thrilled to share our Attendance Management System project, developed using Oracle SQL! I contributed by working on implementing the attendance statistics procedure It was a valuable experience collaborating with the team and applying our SQL skills to build a functional and efficient solution. Huge thanks to everyone for their hard work and dedication! 🙌🚀

**Comment by Jospin20** [comment]
A boost to my skills always when i work in groups and vice versa.
we did a wonderful work guys, appreciated is everyone's participation, keep that courage.

**Comment by ericmaniraguha** [comment]
That is a great work and i notice that **@ishimwedavid4**5 has not contributed to the task.

**Pull request merged**